JavaFXUserInterface/src/main/java/controllers/AssetTypeController.java

```diff
@@ -111,13 +119,110 @@ public void fillAssetTypeTable() {
         return assetTypeList;
     }
 
+    /**
+     *
+     */
+    public void generateThumbnails(ObservableList<AssetTypeList> assetTypeList) {
+        if (assetTypes != null) {
+            for (AssetTypeList assetType : assetTypeList) {
+                Pane pane = new Pane();
+                pane.setOnMouseClicked(event -> uiUtilities.changeScene("/AssetTypeInfo", assetType, pane.getScene()));
+                pane.getStyleClass().add("assetTypePane");
+
+                Text assetTypeName = new Text(assetType.getName());
+                assetTypeName.getStyleClass().add("assetTypeName");
+                Label okLabel = new Label("OK");
+                Label advisoryLabel = new Label("Advisory");
+                Label cautionLabel = new Label("Caution");
+                Label warningLabel = new Label("Warning");
+                Label failedLabel = new Label("Failed");
```

Comment:
I think it would be good if either the values in the colored boxes or the labels were bold to create a differentiation between the labels and the values.

Reply:
done